src/NuGet.Services.Metadata.Catalog.Monitoring/Status/PackageMonitoringStatusService.cs

```diff
@@ -137,12 +177,27 @@ private string GetPackageFileName(FeedPackageIdentity package)
                 $"{idString}.{versionString}.json";
         }
 
-        private Task<PackageMonitoringStatus> GetPackageAsync(Storage storage, FeedPackageIdentity package, CancellationToken token)
+        private FeedPackageIdentity ParsePackageUri(Uri packageUri)
+        {
+            var uriSegments = packageUri.Segments;
```

Comment:
>Segments [](start = 41, length = 8)

Could you provide an example URI in a comment here?

Reply:
good idea